### PR TITLE
Allow webadmins to reload php-fpm

### LINKED ISSUE #26: Allow webadmins to reload php-fpm
I was working on [this issue](https://github.com/carlalexander/debops-wordpress/issues/102). I realized that you probably want to restart php-fpm to erase the opcache. Do you think we could do like `debops.nginx` and give the `webadmins` group the ability to reload it without the sudo password?

### PR DESCRIPTION
**Pull request by carlalexander**
Fixes #26

### CODE REVIEW

**Comment by carlalexander** [comment]
Hmmm the test failed, but I'm not sure I understand why.

**Comment by drybjed** [comment]
@carlalexander No idea either, I'll try running the `master` branch on Travis to see if it's related to the PR or not.

**Comment by drybjed** [comment]
The error might be related to [this Ansible issue](https://github.com/ansible/ansible/issues/14578).

**Comment by carlalexander** [comment]
Do we hold off the merge?

**Comment by drybjed** [comment]
@carlalexander I've found the issue - you cannot use `-` character in the `sudo` command aliases. Change that alias to `PHP_FPM_RELOAD`, that should do the trick. The reason for this weird error on Travis is that `sudo` cannot parse its configuration properly and exits with an error.

**Comment by carlalexander** [comment]
Fixed it! Tests are green now 😂

File: templates/etc/sudoers.d/php-fpm_webadmins.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+
+Cmnd_Alias PHP_FPM_RELOAD = /etc/init.d/php{{ php__version }}-fpm reload
```
Comment (drybjed) [inline templates/etc/sudoers.d/php-fpm_webadmins.j2]: Perhaps add the command for the `systemd` systems as well:
```
Cmnd_Alias PHP_FPM_RELOAD = /etc/init.d/php{{ php__version }}-fpm reload, \
                            systemctl reload php{{ php__version }}-fpm.service
```

**Comment by carlalexander** [comment]
Just added it 😄

**Comment by carlalexander** [comment]
This error seems unrelated to the PR.

**Comment by drybjed** [comment]
Sorry, it might be my fault this time. In the `sudoers` file, try changing the `, \` to `,\`, perhaps this will make `sudo` cooperate.

**Comment by carlalexander** [comment]
Still broken 😢

**Comment by drybjed** [comment]
@carlalexander OK, in that case put all of the commands in one line, that should do it. Maaybe it's a Tab requirement similar to Makefiles, but I'm not sure.

**Comment by carlalexander** [comment]
Using the same line doesn't work either. It should according to [this](https://askubuntu.com/questions/246455/how-to-give-nopasswd-access-to-multiple-commands-via-sudoers). It's worth noting that the error is with the next step `Generate php.ini configuration`.

**Comment by carlalexander** [comment]
Any idea why this keeps failing? It's not doing it at the step that I modified.

**Comment by drybjed** [comment]
@carlalexander Found the issue, you have to specify absolute paths to executables in `sudoers` configuration files. The error you saw on Travis was related to wrong `sudo` configuration and not the next step - remember that each Ansible task is run through `sudo`, if `sudoers` is misconfigured, `sudo` will fail.

**Comment by carlalexander** [comment]
Oops! Thanks for the catch!

**Pull request merged**